acquia_lift.admin.wizard.inc

```diff
@@ -248,9 +248,6 @@ function acquia_lift_personalize_campaign_wizard_targeting_alter(&$form, &$form_
   // Acquia Lift Target agents allow for audience designation to be used across
   // the agent.
   $agent_data = $form['#agent'];
-  if ($agent_data->plugin !== 'acquia_lift_target') {
-    return;
-  }
```

Comment:
Yes because the entire alteration doesn't get run now unless it is an acquia_lift_target campaign so this would be a duplicate check.  Since its the only agent that will be editable from Acquia Lift anyway it seemed easiest.